### PR TITLE
implement basic furniture service

### PR DESCRIPTION
**Pull request by mingxin-yu**
Implement basic furniture service.

### CODE REVIEW

File: src/app/furniture.service.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+@Injectable({providedIn: 'root'})
+export class FurnitureService {
+  // URL to web api
+  private static FURNITURES_URL = 'api/furnitures';
+  httpOptions = {
+    headers: new HttpHeaders({'Content-Type': 'application/json'})
+  };
+
+  constructor(
```
Comment (ghost) [inline src/app/furniture.service.ts:16]: put on the same line

File: src/app/furniture.service.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    private http: HttpClient) {}
+
+  getFurnitures(): Observable<Furniture[]> {
+    return this.http.get<Furniture[]>(FurnitureService.FURNITURES_URL)
+      .pipe(
```
Comment (ghost) [inline src/app/furniture.service.ts]: same line

File: src/app/furniture.service.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+  /**
+   * Handle Http operation that failed.
+   * Let the app continue.
+   * @param operation - name of the operation that failed
+   * @param result - optional value to return as the observable result
+   */
+  private handleError<T>(operation = 'operation', result?: T) {
+    return (error: any): Observable<T> => {
+
```
Comment (ghost) [inline src/app/furniture.service.ts]: remove this empty line

**Review by ghost — APPROVED** [review]

File: src/app/furniture.service.ts (hunk 1 above)
Comment (mingxin-yu) [reply src/app/furniture.service.ts:16]: Fixed

File: src/app/furniture.service.ts (hunk 2 above)
Comment (mingxin-yu) [reply src/app/furniture.service.ts]: fixed

File: src/app/furniture.service.ts (hunk 3 above)
Comment (mingxin-yu) [reply src/app/furniture.service.ts]: fixed

**Pull request merged**